# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/0ad/hotfix-root.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- 0ad-0.27.0/build/workspaces/update-workspaces.sh.vanilla	2025-03-18 17:18:00.731738026 +0200
+++ 0ad-0.27.0/build/workspaces/update-workspaces.sh	2025-03-18 17:18:30.425383600 +0200
@@ -2,7 +2,7 @@
 
 if [ "$(id -u)" = "0" ]; then
 	echo "Running as root will mess up file permissions. Aborting ..." 1>&2
-	exit 1
+	#exit 1
 fi
 
 die()
